cluster/charts/crossplane/templates/rbac-manager-managed-clusterroles.yaml

```diff
@@ -101,7 +101,7 @@ rules:
   verbs: ["*"]
 - apiGroups:
   - pkg.crossplane.io
-  resources: [providers, configurations, providerrevisions, configurationrevisions]
+  resources: [locks, providers, configurations, providerrevisions, configurationrevisions]
```

Comment:
any ideas on the historical context for why `locks` wasn't included? was that oversight or any evidence that it was intentional?

Reply:
I can only speak to the git history: the definition comes b5f01da6aaf786c0f7a6161e6e8c38796107d7f2 which was November 2020. On Nov 25, 2022, cc9bad7537597547005541a713e0a80ebd6c533c introduced the Lock API. @negz may not know, but my git archeology attempt didn't yield an answer to this question. That said, those who might need to do lock surgery like have cluster admin (which is not the case for the managed control planes).

Reply:
My guess is probably oversight. It's not something we think about much. It's mostly Crossplane machinery - humans reading and writing it is an edge case.